app_dart/lib/src/request_handlers/github/webhook_subscription.dart

```diff
@@ -430,6 +430,9 @@ class GithubWebhookSubscription extends SubscriptionHandler {
       // See https://github.com/flutter/flutter/wiki/Plugin-Tests for discussion
       // of various plugin test types and locations.
       if (filename.endsWith('_test.dart') ||
+          // Test files in package-specific test folders.
+          filename.contains('packages/packages/go_router/test_fixes/') ||
```

Comment:
The `packages/packages/` part of these parts unnecessarily ties these patterns to the specific repo structure; we should just start with the package name.